src/index.js

```diff
@@ -36,22 +25,88 @@ btnDeleteAll.setAttribute('class', 'delete-all');
 btnDeleteAll.textContent = 'Clear all completed';
 container.append(btnDeleteAll);
 
+const addTask = (text) => {
+  const newTask = {
+    text,
+    completed: false,
+    index: todoItems.length + 1,
+  };
+  todoItems.push(newTask);
+  localStorage.setItem('todoItems', JSON.stringify(todoItems));
+};
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, according to the requirements, the new functionality should go inside a separate JavaScript file, please consider doing so! 🤓 🔧 

![image](https://user-images.githubusercontent.com/77069011/222596027-fa625eef-b79a-4cea-a0b9-fee966c52a3a.png)

Reply:
For this suggested change I have created another file todo.js and defined my Array there and above function referred by you in the todo.js and imported the module in index.js file. Screen shots attached below for your kind reference. 
![image](https://user-images.githubusercontent.com/117091923/222693801-513f5c6b-1d0a-4284-bbf6-15eb3c1c55ba.png)
![image](https://user-images.githubusercontent.com/117091923/222693951-42a7d18e-e8a7-4c3e-995b-6aa6e9b5342d.png)